reflekt-plugin/src/test/resources/org/jetbrains/reflekt/plugin/compiler/code-gen/general-calling/reflekt/functions/functions-6.kt

```diff
@@ -1,9 +1,9 @@
 // FILE: TestCase.kt
 import org.jetbrains.reflekt.Reflekt
-import org.jetbrains.reflekt.test.helpers.checkFinctionsCallResult
+import org.jetbrains.reflekt.test.helpers.checkFunctionsCallResult
 import org.jetbrains.reflekt.test.common.*
 
-fun box(): String = checkFinctionsCallResult(
+fun box(): String = checkFunctionsCallResult(
```

Comment:
I don't see the difference... what was changed?

Reply:
A typo…